collie/cross_validation.py

```diff
@@ -299,7 +302,8 @@ def _stratified_split_parallel_worker(interactions: BaseInteractions,
                                       idxs_to_split: Iterable[Any],
                                       test_p: float,
                                       seed: int,
-                                      force_split: bool) -> np.array:
+                                      force_split: bool,
+                                      opposite_type: Literal['item', 'user'] = 'user') -> np.array:
```

Comment:
Why does this function take `opposite_type` as an input but then recompute it in the line below?

Reply:
OOPS!